src/sentry/conf/server.py

```diff
@@ -464,6 +465,7 @@ def env(key, default='', type=None):
 CELERY_QUEUES = [
     Queue('alerts', routing_key='alerts'),
     Queue('auth', routing_key='auth'),
+    Queue('commits', routing_key='commits'),
```

Comment:
It will get picked up there, and I'm not completely sure we need to add it to the federation list unless we expect this to be heavily populated.

https://github.com/getsentry/ops/blob/master/cookbooks/getsentry/recipes/queue.rb#L86